src/download/exchangeDownloader.ts

```diff
@@ -297,28 +319,28 @@ export async function getSpecificApi(
  * removes all the version specific information from the returned object.
  *
  * @param query - Exchange search query
- * @param [deployment] - RegExp matching the desired deployment targets
  *
  * @returns Information about the APIs found.
  */
-export async function search(
-  query: string,
-  deployment?: RegExp
-): Promise<RestApi[]> {
-  if (deployment) {
-    ramlToolLogger.warn(DEPLOYMENT_DEPRECATION_WARNING);
-  }
-
+export async function search(query: string): Promise<RestApi[]> {
```

Comment:
We're planning to search based on the API `SDK Type` and `Visibility` categories/types correct? Previously we queried by name... How does it look now when we call `search` function in the SDKs?

Reply:
For node sdk, it will be 
`
await download.search(
      'category:Visibility = "External" category:"SDK Type" = "Commerce"'
    );
`

For isomorphic sdk, it will be 
`
await download.search(
      'category:Visibility = "External" category:"SDK Type" = "Commerce"'  category:"SDK Type" = "Isomorphic"'
    );
`